src/html/userSurvey.popupNoPrompt.html

```diff
@@ -3,18 +3,42 @@
     <head>
         <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
         <meta content="utf-8" http-equiv="encoding">
+        <style>
+            /* May need to add font bundling or revisit the font choice for cross-browser compatibility. */
+            body {
+                font-family: sans-serif;
+            }
+
+            .icon {
+                max-width: 50px;
+                max-height: 50px;
+                float: left;
+                padding-right: 10px; 
+            }
+        </style>
     </head>
     <body>
-        <div id="content" class="paragraph" style="width: 300px; padding-right: 25px; padding-left: 25px;">
+        <div style="display: flex; max-width: 280px; padding: 10px;">
+            <img id="icon" class="icon" style="display: none;">
+            <div id="content" class="paragraph" style="padding-left: 10px; padding-right: 10px;"></div>
         </div>
         <script>
```

Comment:
Hm, I'm surprised that the default CSP rules allow inline script:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_Security_Policy#default_content_security_policy

Reply:
They don't! We use script hashes for these popup inline scripts.